dht/range_streamer.cc

```diff
@@ -288,22 +286,26 @@ future<> range_streamer::stream_async() {
                             _nr_total_ranges - remaining, _nr_total_ranges, _reason, percentage);
                 };
                 try {
+                  if (range_vec.size() <= nr_ranges_per_stream_plan) {
+                    do_streaming(std::move(range_vec));
```

Comment:
Same here,  we might range_vec after it is moved.

Reply:
fixed in v3